tapir/wirgarten/templates/wirgarten/product_cfg/period_product_cfg.html

```diff
@@ -0,0 +1,17 @@
+
+{% load django_bootstrap5 %}
+{% load i18n %}
+{% load static %}
```

Comment:
you don't need to load all tag libraries, only the one you use (bootstrap in this case)

Reply:
It seem to me I need those, otherwise tags i use are not working.